otter/convergence/gathering.py

```diff
@@ -138,6 +141,20 @@ def get_scaling_group_servers(tenant_id, group_id, now,
     yield do_return(servers)
 
 
+def get_all_stacks(stack_tag=None, batch_size=100):
```

Comment:
likewise `batch_size` is an unused argument. I'd rather not give the impression that batching is supported when it's not